compiler/rustc_codegen_cranelift/src/base.rs

```diff
@@ -737,6 +737,17 @@ fn codegen_stmt<'tcx>(
                         CValue::const_val(fx, fx.layout_of(fx.tcx.types.usize), ty_size.into());
                     lval.write_cvalue(fx, val);
                 }
+                Rvalue::NullaryOp(NullOp::AlignOf, ty) => {
```

Comment:
Please merge all these arms with the `SizeOf` one by only matching on the `AlignOf`/`SizeOf` part for the actual value computation.